pkg/oc/clusterup/coreinstall/kubelet/run_kubelet.go

```diff
@@ -31,6 +31,7 @@ func NewKubeletRunConfig() *KubeletRunConfig {
 		ContainerBinds: []string{
 			"/var/log:/var/log:rw",
 			"/var/run:/var/run:rw",
+			"/var/lib/kubelet:/var/lib/kubelet:rw",
```

Comment:
why?

Reply:
Should have been in a separate commit, looks like I accidentally saved it here.